extensions/wikia/Forum/ForumHooksHelper.class.php

```diff
@@ -1,6 +1,39 @@
 <?php
 
 class ForumHooksHelper {
+
+	static public function discussionNavigationOverride(
+		&$title, &$request, &$ignoreRedirect, &$target, Article &$article
+	) {
+		if ( ForumHelper::isForum() &&
+		     DiscussionsHelper::areForumsArchivedAndDiscussionsEnabled()
+		) {
+			$ignoreRedirect = false;
+			$target = self::redirectToDiscussions( $title );
+		}
+
+		return true;
+	}
+
+	private static function redirectToDiscussions(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1142)